nitro-src/claim.ts

```diff
@@ -14,7 +14,11 @@ export function claim(
 ) {
   if (initialTargetOutcome.length !== initialHoldings.length) throw Error;
   const updatedTargetOutcome: Exit = [];
-  let updatedHoldings = initialHoldings;
+  // We want to create a clone of the original values to prevent mutation
+  const updatedHoldings = initialHoldings.map((h) => BigNumber.from(h));
+  const updatedGuaranteeOutcome: Exit = JSON.parse(
+    JSON.stringify(initialGuaranteeOutcome)
```

Comment:
This is just an easy way to clone any objective (without adding lodash as a dependency :P )